gm4/skin_cache.json

```diff
@@ -1001,11 +990,11 @@
       "gm4_metallurgy:band/aluminium"
     ],
     "gm4_zauber_liquids": [
+      "gm4_potion_liquids:liquids/healing",
+      "gm4_potion_liquids:liquids/regeneration",
       "gm4_potion_liquids:liquids/poison",
       "gm4_potion_liquids:liquids/harming",
       "gm4_potion_liquids:liquids/strength",
-      "gm4_potion_liquids:liquids/regeneration",
-      "gm4_potion_liquids:liquids/healing",
       "gm4_potion_liquids:liquids/leaping",
       "gm4_potion_liquids:liquids/swiftness"
```

Comment:
Would it be difficult to sort this list?

Reply:
It should be pretty easy. I just didn't want to do it in this PR. Ill add it though since it's tiny